homework/index.js

```diff
@@ -27,21 +27,159 @@
         elem.setAttribute(key, value);
       }
     });
+
     return elem;
   }
 
+  function sortList() {
+    let i;
+    let switching;
+    let b;
+    let shouldSwitch;
+    const list = document.getElementById('select');
+    switching = true;
+    while (switching) {
+      switching = false;
+      b = list.getElementsByTagName('option');
+      for (i = 0; i < b.length - 1; i++) {
+        shouldSwitch = false;
+        if (b[i].innerHTML.toLowerCase() > b[i + 1].innerHTML.toLowerCase()) {
+          shouldSwitch = true;
+          break;
+        }
+      }
+      if (shouldSwitch) {
+        b[i].parentNode.insertBefore(b[i + 1], b[i]);
+        switching = true;
+      }
+    }
+  }
+
+  // fetch url contributor
+  function fetchContributorsUrl(contributorsUrl) {
+    fetchJSON(contributorsUrl, (err, contributor) => {
+      if (err) {
+        const root = document.getElementById('root');
+        createAndAppend('div', root, { text: err.message, class: 'alert-error' });
+      } else {
+        for (let j = 0; j < contributor.length; j++) {
+          const nameOfContributor = contributor[j].login;
+          const numberOfContributor = contributor[j].contributions;
+          const avatarUrl = contributor[j].avatar_url;
+          const cont = document.getElementById('cont');
+          const avatarContainer1 = createAndAppend('div', cont, {
+            class: 'avatar_div',
+          });
+          const avatarContainer2 = createAndAppend('div', cont, { class: 'avatar_div data' });
```

Comment:
Avoid variables numbering by using different and descriptive names, like: `avatarContainer1` -> `avatarContainer` and `avatarContainer2` -> `contributorNameContainer`?